iis-core/src/test/java/eu/dnetlib/iis/core/common/AvroAssertTestUtil.java

```diff
@@ -41,5 +43,21 @@ private AvroAssertTestUtil() {
                 avroDatastore.stream().map(T::toString).collect(Collectors.toList()));
 
     }
+    
+    /**
+     * Asserts equality of avro datastore to json datastore ignoring order of records
```

Comment:
Asserts equality of an avro datastore and a json datastore, ignores the order of records.